identity/src/ecdsa.rs

```diff
@@ -102,14 +109,23 @@ impl SecretKey {
         signature.as_bytes().to_owned()
     }
 
-    /// Encode a secret key into a byte buffer.
+    /// Convert a secret key into a byte buffer.
     pub fn to_bytes(&self) -> Vec<u8> {
         self.0.to_bytes().to_vec()
     }
 
     /// Decode a secret key from a byte buffer.
-    pub fn from_bytes(buf: &[u8]) -> Result<Self, DecodingError> {
-        SigningKey::from_bytes(buf)
+    #[deprecated(
+        since = "0.2.0",
+        note = "This method name does not follow Rust naming conventions, use `SecretKey::try_from_bytes` instead"
+    )]
+    pub fn from_bytes(buf: &[u8]) -> Result<SecretKey, DecodingError> {
```

Comment:
I believe this would all git to recognise the diff properly.

```suggestion
    pub fn from_bytes(buf: &[u8]) -> Result<Self, DecodingError> {
```